### PR TITLE
rewrite title and link of podcast

### PR DESCRIPTION
**Pull request by fredel-mamindra**

### CODE REVIEW

File: content/podcasts/11-solution-engineering-from-teradata-and-oracle-to-open source.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 description: "Brian explores what he sees as the drivers and obstacles for enterprises moving and adopting more open source."
 short_text: "Percona’s Hoss (Head of Open Source Strategy) Matt Yonkovit talks to Percona’s SR Director of Solution Engineering on his journey from Teradata and Oracle to Open Source at Percona. Brian explores what he sees as the drivers and obstacles for enterprises moving and adopting more open source."
 date: "2021-03-03"
 podbean_link: "https://percona.podbean.com/e/the-hoss-talks-foss-ep11-talking-oss-linux-oracle-big-business-and-foss-with-brian-walters/"
 youtube_id: "HTrGSfs8g1I"
 speakers:
   - brian_walters
+aliases:
+    - "/podcasts/11/"
+url: "/podcasts/11-solution-engineering-from-teradata-and-oracle-to-open source"
```
Comment (dbazhenov) [inline content/podcasts/11-solution-engineering-from-teradata-and-oracle-to-open source.md:12]: omitted "-" in the last word.

File: content/podcasts/12-database-on-kubernetes-operator-pitr-point-in-time recovery.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 description: "Sergey Pronin (Percona Product Owner for Percona's Kubernetes operators) visits with the HOSS again talking about the latest releases as Percona adds functionality to Percona's MongoDB and MySQL Operators (Also coming soon a PostgreSQL Operator!)."
 short_text: "Sergey Pronin (Percona Product Owner for Percona's Kubernetes operators) visits with the HOSS again talking about the latest releases as Percona adds functionality to Percona's MongoDB and MySQL Operators (Also coming soon a PostgreSQL Operator!)."
 date: "2021-03-04"
 podbean_link: "https://percona.podbean.com/e/the-hoss-talks-foss-ep12-sergey-pronin-returns-to-talk-about-kubernets-operators-and-dbaas/"
 youtube_id: "hHxLBRvCZeg"
 speakers:
   - sergey_pronin
+aliases:
+    - "/podcasts/12/"
+url: "/podcasts/12-database-on-kubernetes-operator-pitr-point-in-time recovery"
```
Comment (dbazhenov) [inline content/podcasts/12-database-on-kubernetes-operator-pitr-point-in-time recovery.md:12]: omitted "-" in the last word.

File: content/podcasts/2-mysql-backups-database-recovery-best practices.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 description: "We cover details about Walter's latest blog outlining MySQL Backup best practices."
 short_text: "The Hoss talks with Walter Garcia remote dba expert for Percona about how to ensure you are getting good and consistent MySQL backups. We cover details about Walter's latest blog outlining MySQL Backup best practices."
 date: "2021-02-16"
 podbean_link: "https://percona.podbean.com/e/the-hoss-talks-foss-ep02-talking-mysql-backup-best-practices-with-walter-garcia/"
 youtube_id: "AJaze6ehqm0"
 speakers:
   - walter_garcia
+aliases:
+    - "/podcasts/2/"
+url: "/podcasts/2-mysql-backups-database-recovery-best practices"
```
Comment (dbazhenov) [inline content/podcasts/2-mysql-backups-database-recovery-best practices.md:12]: omitted "-" in the last word.

File: content/podcasts/45-mattermost-corye-hulen-open-source-ospo-percona.md [hunk 4]
```diff
@@ -1,12 +1,15 @@
...
 short_text: "Engaging users and contributors is key to building a strong and vibrant open source community. At Mattermost, the team spends a lot of time ensuring their community can be successful by making the contribution process as easy as possible and providing a welcoming environment. Corey Hulen the CTO and Co-Founder of Mattermost joins the HOSS to talk about his journey to the open source space and how he has been helping to build a culture internally that promotes and embraces the community as key members of the engineering team. Listen in and get tips on how to grow your contributor communities as well as learn about the cool things happening at Mattermost. Whether you are in your company's OSPO (Open source Program Office), DevRel team, or an open source maintainer this is a great talk to learn about some cool ideas and best practices."
 date: "2021-11-05"
 podbean_link: "https://percona.podbean.com/e/the-hoss-talks-foss-45-building-a-strong-open-source-community-corey-hulen-cto-of-mattermost/"
 youtube_id: "5sYSKxF97OM"
 speakers:
   - corey_hulen
+aliases:
+    - "/podcasts/45-mattermost-corye-hulen-open-source-ospo-percona/"
```
Comment (dbazhenov) [inline content/podcasts/45-mattermost-corye-hulen-open-source-ospo-percona.md:11]: It doesn't change the URL, it doesn't need to.

Comment (fredel-mamindra) [reply content/podcasts/45-mattermost-corye-hulen-open-source-ospo-percona.md:11]: there was misspelling of "corey"

**Pull request merged**